server/constants/tours/eyeSpy/may-2022.json

```diff
@@ -0,0 +1,98 @@
+{
+  "title": "Eye Spy Gallery Scavenger Hunt",
+  "subtitle": "",
+  "heroImageInvno": "BF331",
+  "heroImageStyle": {
+    "overlay": {
+      "backgroundColor": "rgba(0, 0, 0, 0.4)"
+    }
+  },
+  "description": "<p>Can you spy all of these works in our galleries? Find a member of the Barnes team wearing a white apron to mark off your sheet when you find them and turn in the completed sheet for a prize! </p>",
+  "body": {
+    "from": 0,
+    "size": 25,
+    "_source": [
+      "id",
+      "title",
+      "people",
+      "medium",
+      "imageOriginalSecret",
+      "imageSecret",
+      "ensembleIndex",
+      "objRightsTypeId",
+      "onview",
+      "invno",
+      "image",
+      "curatorialApproval",
+      "shortDescription",
+      "nationality",
+      "birthDate",
+      "deathDate",
+      "artistPrefix",
+      "artistSuffix",
+      "culture",
+      "displayDate",
+      "medium",
+      "dimensions",
+      "creditLine",
+      "longDescription",
+      "bibliography",
+      "exhHistory",
+      "publishedProvenance"
+    ],
+    "query": {
+      "bool": {
+        "filter": {
+          "terms": {
+            "invno": [
+              "bf136",
+              "bf357",
+              "bf2554",
+              "bf80",
+              "a230",
+              "a126",
+              "a108",
+              "bf278",
+              "a250"
+            ]
+          }
+        }
+      }
+    }
```

Comment:
Was this tour working when you ran it locally? 

If you check out the template document, you will see that this query here needs to be populated with the inventory numbers of the objects to be included in the tour: https://github.com/BarnesFoundation/Barnes-Online-Collection/blob/1a0199470aac8b7cbcf8cd958f24d055faba9c99/server/constants/tours/template.jsonc#L64-L78.

Reply:
oops 🤦 thanks!